### ISSUE TITLE
Results column names don't match with data

### ISSUE THREAD

**Issue by lis131** [OpenMendel/TrajGWAS.jl#44]
Hello,
Thanks for developing this great tool.

I was trying to follow the instructions using the example datasets. When I try to run with the interaction term using the formula:  testformula=@formula(y ~ snp + snp & sex), the got the following results. I'm not sure which columns those last 3 numbers match to. I compared with the results without interaction term, seems they represent those pvalues? thanks

|   chr |   pos |    snpid |     varid | allele1 | allele2 |    hwepval |         maf |   infoscore | betapval | taupval | jointpval |
| Int64 | Int64 | String15 |  String15 | String1 | String1 |    Float64 |     Float64 |     Float64 |  Missing | Missing |   Missing |
|-------|-------|----------|-----------|---------|---------|------------|-------------|-------------|----------|---------|-----------|
|     1 |  1001 | RSID_101 | SNPID_101 |       A |       G |   0.792312 |    0.518204 |    0.626591 |  missing | missing |   missing |
|     1 |  2000 |   RSID_2 |   SNPID_2 |       A |       G |  1.7332e-7 | 3.54836e-20 | 7.09672e-20 |  missing | missing |   missing |
|     1 |  2001 | RSID_102 | SNPID_102 |       A |       G | 1.71603e-7 | 4.43481e-20 | 8.86961e-20 |  missing | missing |   missing |
|     1 |  3000 |   RSID_3 |   SNPID_3 |       A |       G |   0.184132 |    0.486155 |      0.2671 |  missing | missing |   missing |
|     1 |  3001 | RSID_103 | SNPID_103 |       A |       G |   0.184132 |    0.486155 |      0.2671 |  missing | missing |   missing |

**Comment by kose-y** [comment]
You are correct, the last three numbers are betapval, taupval, and jointpval; hwepval, maf, infoscore are not printed in this case. Thank you for pointing this out.
